package.json

```diff
@@ -8,14 +8,16 @@
     "start:android": "npx expo start --android",
     "start:ios": "npx expo start --ios",
     "prebuild": "rm -rf android; rm -rf ios; rm -rf node_modules; yarn; npx expo prebuild --clean --no-install",
+    "build:all": "eas build --platform all",
     "build:android": "eas build --platform android",
     "build:ios": "eas build --platform ios",
     "build:dev-android": "eas build --profile development --platform android",
     "build:dev-ios": "eas build --profile development --platform ios",
     "build:dev-ios-simulator": "eas build --profile development-simulator --platform ios",
     "build:local-ios-prod": "npx expo run:ios -d --configuration Release",
-    "submit:android": "eas submit --platform android",
-    "submit:ios": "eas submit --platform ios",
+    "submit:all": "eas submit --platform all --latest",
+    "submit:android": "eas submit --platform android --latest",
+    "submit:ios": "eas submit --platform ios --latest",
```

Comment:
what if it would be not `latest`? what are other values or use cases?

Reply:
`latest` is a flag added to send the latest build created in eas to the stores. I removed this flag because I don't think it is needed in local. Because in local we have the possibility to choose the desired build.